contracts/Staking.sol

```diff
@@ -19,13 +19,13 @@ contract Staking is AccessControlEnumerable {
 
     function setPermission(
         address sender
-    ) external onlyRole(STAKING_ADMIN_ROLE) {
+    ) external onlyRole(DEFAULT_ADMIN_ROLE) {
```

Comment:
why change this? unrelated change.

Reply:
Unrelated to the issue created but a change is required to do since only the DEFAULT_ADMIN_ROLE can grant/revoke roles so using `STAKING_ADMIN_ROLE` won't be feasible. @hrishikeshio

Reply:
Remove `setPermission` and `revokePermission`